### DISCUSSION TITLE
How to share a locally mounted MinIO S3 server using Mountain Duck with colleagues?

### DISCUSSION THREAD

**Discussion by Huumour93** [iterate-ch/cyberduck#14541] [Q&A]
I have mounted my Minio S3 server as a local disk on my Windows laptop using Mountain Duck, but I need to share the mounted disk with my colleagues. What is the best way to share this mounted disk with multiple users on a network? Are there any security concerns to consider when sharing a mounted disk in this way?

**Comment by tatze96** [comment]
Mountain  Duck mounts can't be shared within the local network as they are bound and limited to the active user session.  Because of this, each user that needs to access the server must mount it in their own user session.

If you only want to share the files within a folder, you can use one of the available sharing options depending on the used protocol.
For more information on the available sharing options, refer to our [wiki page](https://docs.cyberduck.io/mountainduck/share/).

**Reply by Huumour93** [reply]
Thank you.
